Jint/Collections/Tuple.cs

```diff
@@ -0,0 +1,20 @@
+namespace Jint.Collections
+{
+    public readonly struct Tuple<A,B>
```

Comment:
internal probably

Reply:
I wanted to use it in public API (which I haven't flushed out yet)